monkey/monkey_island/cc/next_ui/src/_components/app-nav/app-menu/style.ts

```diff
@@ -0,0 +1,7 @@
+export const appTabsContainer = {
+    width: '100%',
+    height: '100% !important',
+    '& .MuiTabs-flexContainer': {
+        height: '100% !important'
```

Comment:
I'm not seeing the `!important` selector making any difference (testing in Chrome) for either of these instances. What problem does it overcome?